darts/tests/test_timeseries.py

```diff
@@ -165,6 +165,31 @@ def test_integer_range_indexing(self):
         # getting index for idx should return i s.t., series[i].time == idx
         assert series.get_index_at_point(16) == 3
 
+    def test_grom_group_dataframe(self):
```

Comment:
Small typo in the name of the function
```suggestion
    def test_from_group_dataframe(self):
```